tests/cache_sorted_set_tests.rs

```diff
@@ -0,0 +1,311 @@
+use uuid::Uuid;
+
+use momento::requests::cache::sorted_set_fetch_by_rank::SortOrder::{Ascending, Descending};
+use momento::requests::cache::sorted_set_fetch_by_rank::SortedSetFetchByRankRequest;
+use momento::requests::cache::sorted_set_fetch_by_score::SortedSetFetchByScoreRequest;
+use momento::response::cache::sorted_set_fetch::SortedSetFetch;
+
+use crate::cache_test_state::TEST_STATE;
+
+mod cache_test_state;
+
+#[tokio::test]
+async fn sorted_set_put_element_happy_path() {
```

Comment:
so, one thing I wanted to sync with you on, it seems like in a lot of rust code the tests are just added to the same file as the actual code that is being tested. idk if we have prior art on that in this repo or if that is sane, or if for us it makes more sense to keep them separate especially since they are integration tests. any thoughts?

Reply:
I honestly hadn't thought about that. I just saw we had a tests directory with some integration tests in it and my Java instincts kicked in. I would need to do an experiment to see if the global test state would still work the same way, but I'm good with adding them to the same file. My only concern would be if it would become hard to read if the cache client were filled with doctests and integration tests.

Reply:
k, will fish around for opinions on the backend team... on this one i could make a case either way.